torsiondrive/td_api.py

```diff
@@ -236,7 +236,8 @@ def current_state_json_load(json_state_dict):
 def next_jobs_json_dict(next_jobs):
     """ Dump the next_jobs dictionary to a json file """
     json_next_jobs = {}
-    for grid_id, new_job_list in next_jobs.items():
+    for grid_id in sorted(next_jobs):
+        new_job_list = next_jobs[grid_id]
```

Comment:
Since dictionaries are ordered now, it is better the keys sorted before returning to maximize reproducibility.